pom.xml

```diff
@@ -76,7 +76,7 @@
         <dep.ratis.version>2.2.0</dep.ratis.version>
         <dep.errorprone.version>2.18.0</dep.errorprone.version>
         <dep.guava.version>32.1.0-jre</dep.guava.version>
-        <dep.jackson.version>2.11.0</dep.jackson.version>
+        <dep.jackson.version>2.17.2</dep.jackson.version>
```

Comment:
why not 2.18.0?

Reply:
> @Mariamalmesfer @elharo Maybe we can update to the one version before the very latest which is `2.17.2` what do you think?

We can proceed with either version, but since Reetika mentioned updating to 2.17.2, Let me know what you think.

Reply:
@elharo Since version 2.18.0 is the very latest, I was thinking we could use the previous version instead to avoid potential issues with the latest release.

Reply:
@elharo, @agrawalreetika pointed out the option to update to 2.17.2 instead of the latest 2.18.0. Please advise on the preferred version.